util/stmtsummary/variables_test.go

```diff
@@ -14,108 +14,108 @@
 package stmtsummary
 
 import (
-	. "github.com/pingcap/check"
+	"testing"
+
 	"github.com/pingcap/tidb/config"
+	"github.com/stretchr/testify/require"
 )
 
-var _ = Suite(&testVariablesSuite{})
-
-type testVariablesSuite struct {
-}
-
-func (s *testVariablesSuite) TestSetInVariable(c *C) {
+func TestSetInVariable(t *testing.T) {
+	t.Parallel()
 	sv := newSysVars()
 	st := sv.getVariable(typeMaxStmtCount)
-	c.Assert(st, Equals, int64(config.GetGlobalConfig().StmtSummary.MaxStmtCount))
+	require.Equal(t, int64(config.GetGlobalConfig().StmtSummary.MaxStmtCount), st)
 
 	err := sv.setVariable(typeMaxStmtCount, "10", false)
-	c.Assert(err, IsNil)
+	require.NoError(t, err)
 	st = sv.getVariable(typeMaxStmtCount)
```

Comment:
It seems that we don't need the var `st`. How about the following code?
```
require.Equal(t, int64(10), sv.getVariable(typeMaxStmtCount))
```

Reply:
thank you for your suggestion.
please forgive me，but i think keep the var `st` there is more clear and more consistent with the err part
```go
	err = sv.setVariable(typeEnable, "OFF", false)
	require.NoError(t, err)
```

another reason is that this is a migrate, less change may let the whole job faster done and less mistake injected.

Reply:
OK.